### ISSUE TITLE
in manual search: Group same source together when column is sorted by Timestamped

### ISSUE THREAD

**Issue by Rexadev** [jacquesh/foo_openlyrics#382]
https://github.com/user-attachments/assets/4e77695a-6a73-4cbb-bf6b-950875490895

**Comment by ngs428** [comment]
This kind of goes with #398 i just submitted.  

Have the results sorted by time stamped at the top and grouped by source.  Sources would be in the same order as they are in the sources list.  Obviously you want your best sources at the top of that list.

**Comment by Rexadev** [comment]
@ngs428  I suggest option 1st sort by Synced / Unsynced - 2 options tick boxes or have both listed with a line divided them into 2 groups
then group them by sources in order in config

**Comment by ngs428** [comment]
@Rexadev I read this a couple times and think I know what you are requesting.

The 2nd method you mentioned is the same as what I said above, but adding a line to separate.  I don’t mind the line, but it should be clear what is and isn’t time stamped.

**Comment by jacquesh** [comment]
The way this would work would be to have a stack of sort columns that you push onto when you sort by a new column.

For example: If you sorted by title then it would be sorted only by title. If you then sorted by album it should sort first by album, and then by title when there are multiple tracks on the same album. If you then sorted by artist it would should sort first by artist, then by album for entries with the same artist, then by title for entries with the same artist and album, etc.

I'm not entirely sure if the native windows list view supports this but...thats what I think the ideal behaviour would be for a table with sortable columns in general.

**Comment by ngs428** [comment]
Yes, that makes sense @jacquesh.  It would greatly assist in getting the most wanted results in order.  Also, implementing the other change we talked about (specifying the sources to use in manual mode), will clean it up even more.  

Thanks again for your work on this!

**Comment by Rexadev** [comment]
Always show Metadata and .lrc at top
